.travis.yml

```diff
@@ -3,7 +3,6 @@ node_js:
   - "node"
   - "8"
   - "6"
-  - "4"
```

Comment:
Please combine this with the change to remove v4 support from package.json into a commit that explains why v4 can't be supported.  That commit should precede the changes to the implementation, since it's independent but (presumably) enables the main change.

Reply:
It doesn't enable the main change, I will break this change up!